buildkitd/buildkitd.go

```diff
@@ -561,6 +561,7 @@ func checkConnection(ctx context.Context, address string, opts ...client.ClientO
 		if err != nil {
 			connErrMu.Lock()
 			connErr = err
+			fmt.Println(err.Error())
```

Comment:
Deliberate or leftover debugging? The err handling code above looks similar but doesn't do `fmt.Println`